packages/_site/src/pages/components/progressbar/index.tsx

```diff
@@ -108,7 +108,7 @@ export default function PopoverExample() {
                         <div>
                             <ExampleBlock
                                 title="Danger"
-                                description="Red progress bar."
+                                description="A progress bar only stops and turns red if there's an error."
```

Comment:
A bit of rewording would be good, since the component doesn't automatically change to red / danger if there's an error.